client/src/App.jsx

```diff
@@ -8,19 +11,36 @@ import RelatedItems from './components/RelatedItems.jsx';
 class App extends React.Component {
   constructor(props) {
     super(props);
+    this.state = {
+      products: sampleData.productList,
+      productId: sampleData.productList[0].id
+    }
+  }
+
+  componentDidMount() {
+    utils.getProducts()
+    .then(productList => {
+      this.setState({
+        products: productList,
+        productId: productList[0].id
+      });
+    })
+    .catch(err => {
+      console.log(err);
```

Comment:
Big fan of error messages with some context from us, makes easier to debug. Maybe something like a reference to this file at this line number and stating most likely what went wrong along with the error message.
line 28 `console.log('Async Function Error at App.jsx Line 28; Issue getting products from external API, good chance authorization was not set up or campus code is not correct, here is the error: ', err);`

Probably redundant but as this app of ours gets bigger and flooded with more code, it will be easier with some clear pointers of what went wrong where and probably something goofy we know would have probably happened.